lib/LookupService.h

```diff
@@ -21,6 +21,7 @@
 
 #include <pulsar/Result.h>
 
+#include <boost/optional.hpp>
 #include <memory>
```

Comment:
It's better to include `<pulsar/Schema.h>` here though it does not affect the compilation because the `.cc` files that includes `LookupService.h` already include `Schema.h`. But for this single file, it might make some static code analysis checks fail. 

![image](https://user-images.githubusercontent.com/18204803/211551587-193a12cb-1e1a-4e2e-8354-8179e4b48eb7.png)